docs/src/APIs/ClimaLand.md

```diff
@@ -34,73 +18,13 @@ ClimaLand.LandModel{FT}(
         ClimaLand.Domains.HybridBox,
     },
     Δt;
-    soil = Soil.EnergyHydrology{FT}(
-        domain,
-        forcing,
-        toml_dict;
-        prognostic_land_components = (:canopy, :snow, :soil, :soilco2),
-        additional_sources = (ClimaLand.RootExtraction{FT}(),),
-    ),
-    soilco2 = Soil.Biogeochemistry.SoilCO2Model{FT}(
-        domain,
-        Soil.Biogeochemistry.SoilDrivers(
-            Soil.Biogeochemistry.PrognosticMet(soil.parameters),
-            PrescribedSoilOrganicCarbon{FT}(TimeVaryingInput((t) -> 5)),
-            forcing.atmos,
-        ),
-    ),
-    canopy = Canopy.CanopyModel{FT}(
-        Domains.obtain_surface_domain(domain),
-        (;
-            atmos = forcing.atmos,
-            radiation = forcing.radiation,
-            ground = ClimaLand.PrognosticGroundConditions{FT}(),
-        ),
-        LAI,
-        toml_dict;
-        prognostic_land_components = (:canopy, :snow, :soil, :soilco2),
-    ),
-    snow = Snow.SnowModel(
-        FT,
-        ClimaLand.Domains.obtain_surface_domain(domain),
-        forcing,
-        toml_dict,
-        Δt;
-        prognostic_land_components = (:canopy, :snow, :soil, :soilco2),
-    ),
 ) where {FT}
 ClimaLand.SoilCanopyModel
 ClimaLand.SoilCanopyModel{FT}(
     forcing,
     LAI,
     toml_dict::CP.AbstractTOMLDict,
     domain::Union{ClimaLand.Domains.Column, ClimaLand.Domains.SphericalShell};
-    soil = Soil.EnergyHydrology{FT}(
-        domain,
-        forcing,
-        toml_dict;
-        prognostic_land_components = (:canopy, :soil, :soilco2),
-        additional_sources = (ClimaLand.RootExtraction{FT}(),),
-    ),
-    soilco2 = Soil.Biogeochemistry.SoilCO2Model{FT}(
-        domain,
-        Soil.Biogeochemistry.SoilDrivers(
-            Soil.Biogeochemistry.PrognosticMet(soil.parameters),
-            PrescribedSoilOrganicCarbon{FT}(TimeVaryingInput((t) -> 5)),
-            forcing.atmos,
-        ),
-    ),
-    canopy = Canopy.CanopyModel{FT}(
-        Domains.obtain_surface_domain(domain),
-        (;
-            atmos = forcing.atmos,
-            radiation = forcing.radiation,
-            ground = ClimaLand.PrognosticSoilConditions{FT}(),
-        ),
-        LAI,
-        toml_dict;
-        prognostic_land_components = (:canopy, :soil, :soilco2),
-    ),
 ) where {FT}
 ClimaLand.LandHydrology
 ClimaLand.LandHydrology{FT}(;
```

Comment:
Can make the same changes for this constructor (and `LandSoilBiogeochemistry` below)